src/main/java/net/dv8tion/jda/core/audit/AuditLogKey.java

```diff
@@ -153,6 +155,18 @@
      */
     CHANNEL_TOPIC("topic"),
 
+    /**
+     * Change of the {@link net.dv8tion.jda.core.entities.TextChannel#getSlowmode() TextChannel.getSlowmode()} value.
+     * <br>Only for {@link net.dv8tion.jda.core.entities.ChannelType#TEXT ChannelType.Text}
+     *
+     * <p>Expected type: <b>Integer</b>
+     *
+     * @incubating
+     *         <b>Currently slowmode is an undocumented and unreleased API feature.</b>
```

Comment:
There is no need to make these bold. They are already in their own section of the docs.

Reply:
FYI, jake already asked the community to document slowmode. I would argue its not "official" but still sort of released. Maybe `Slowmode is currently only available through the API and might still be changed in the future`

Reply:
I was not aware of this, I will change (and unbold) the comments as soon as I can.